### DISCUSSION TITLE
Reject by default with GUI not connected (vs deny)

### DISCUSSION THREAD

**Discussion by BetweenTheKeyboardAndChair** [evilsocket/opensnitch#1108] [Q&A]
Is there a reason that "reject" is not one of the options for the default behavior when the GUI is not available?  Can it be added?

I found that when I had the GUI default action "deny", I was seeing all sorts of hangs in the browser as things waited for some timeout.  These seem to go away when "reject" was added.  I suspect making the same change at the non-GUI default would help.

(See related post about trying to find requests denied w/o GUI)

**Comment by gustavo-iniguez-goya** [comment]
Hi @BetweenTheKeyboardAndChair ,

As far as I can tell there's no reason. Probably no one asked for it. I'll review the code, and if is there no stopper I'll add it soon.

**Comment by gustavo-iniguez-goya** [comment]
added!
